src/js/hca/definitions/child.js

```diff
@@ -64,35 +60,35 @@ export const uiSchema = {
   ],
   childFullName: fullNameUI,
   childRelation: {
-    'ui:title': 'Child’s relationship to you'
+    'ui:title': 'Dependent’s relationship to you?'
   },
   childSocialSecurityNumber: _.merge(ssnUI, {
-    'ui:title': 'Child’s social security number'
+    'ui:title': 'Dependent’s Social Security number'
   }),
-  childDateOfBirth: currentOrPastDateUI('Child’s date of birth'),
-  childBecameDependent: _.assign(currentOrPastDateUI('Date child became dependent'), {
+  childDateOfBirth: currentOrPastDateUI('Dependent’s date of birth'),
+  childBecameDependent: _.assign(currentOrPastDateUI('Date they became your dependent?'), {
     'ui:validations': [
       validateDependentDate
     ]
   }),
   childDisabledBefore18: {
-    'ui:title': 'Was child permanently and totally disabled before the age of 18?',
+    'ui:title': 'Was your dependent permanently and totally disabled before the age of 18?',
     'ui:widget': 'yesNo'
   },
   childAttendedSchoolLastYear: {
-    'ui:title': 'If child is between 18 and 23 years of age, did child attend school during the last calendar year?',
+    'ui:title': 'If your dependent is between 18 and 23 years of age, did they attend school during the last calendar year?',
     'ui:widget': 'yesNo'
   },
-  childEducationExpenses: currencyUI('Expenses paid by your dependent child for college, vocational rehabilitation, or training (e.g., tuition, books, materials)'),
+  childEducationExpenses: currencyUI('Expenses your dependent paid for college, vocational rehabilitation, or training (e.g., tuition, books, materials)'),
   childCohabitedLastYear: {
-    'ui:title': 'Did your child live with you last year?',
+    'ui:title': 'Did your dependent live with you last year?',
     'ui:widget': 'yesNo'
   },
   'view:childSupportDescription': {
-    'ui:description': 'Count child support contributions even if not paid in regular set amounts. Contributions can include tuition payments or payments of medical bills.'
+    'ui:description': 'Please count all support contributions even if they weren’t paid in regular and set amounts. Support can include tuition or medical bill payments.'
```

Comment:
Does this pertain to the following question? If so, should it be visually located closer to it?

Reply:
Yeah, it looks like it's grouped with the question above it, which is odd. I'll make a ticket for it.